aloha-io-avro/src/main/scala/com/eharmony/aloha/factory/avro/StdAvroModelFactory.scala

```diff
@@ -43,68 +51,20 @@ object StdAvroModelFactory {
     * @return A Try of a ModelFactory that creates models taking `GenericRecord` instances as
     *         input and returns `com.eharmony.aloha.audit.impl.avro.Score` as output.
     */
+  @deprecated(message = "Prefer ", since = "4.0.1")
```

Comment:
I deprecated this because I would like everyone to use the config objects instead.  Then, someday once this is removed, that can be deprecated and moved to `apply`.